<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>235</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>525</width>
    <height>235</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>525</width>
    <height>235</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Go To Setpoint</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources/nSOTScannerResources.qrc">
    <normaloff>:/nSOTScanner/Pictures/jj_schema.svg</normaloff>:/nSOTScanner/Pictures/jj_schema.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{background-color: rgb(2, 2, 2);}
#line_11{color: rgb(168,168,168)}
#line_12{color: rgb(168,168,168)}
#line_13{color: rgb(168,168,168)}
#line_14{color: rgb(168,168,168)}
#line_15{color: rgb(168,168,168)}
#line_16{color: rgb(168,168,168)}
#line_17{color: rgb(168,168,168)}

QLabel{
color: rgb(168,168,168);
font: bold 10pt;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="biasPntsLine">
    <property name="geometry">
     <rect>
      <x>157</x>
      <y>110</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>500</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="Line" name="line_13">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>61</y>
      <width>3</width>
      <height>139</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_13{color: rgb(168,168,168)}
#line_11{color: rgb(168,168,168)}
#line_12{color: rgb(168,168,168)}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>217</x>
      <y>75</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#a8a8a8;&quot;&gt;(V)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zeroBiasBtn">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>180</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed#zeroBiasBtn{
color: rgb(95,107,166);
background-color:rgb(168,168,168);
border: 1px solid rgb(95,107,166);
border-radius: 5px
}

QPushButton#zeroBiasBtn{
color: rgb(95,107,166);
background-color:rgb(0,0,0);
border: 2px solid  rgb(95,107,166);
border-radius: 5px
}</string>
    </property>
    <property name="text">
     <string>Zero
Bias</string>
    </property>
   </widget>
   <widget class="Line" name="line_16">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>199</y>
      <width>50</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_9{color: rgb(168,168,168);}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>27</x>
      <y>75</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#a8a8a8;&quot;&gt;Bias Setpoint:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="blinkBtn">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed#blinkBtn{
color: rgb(95,107,166);
background-color:rgb(168,168,168);
border: 1px solid rgb(95,107,166);
border-radius: 5px
}

QPushButton#blinkBtn{
color: rgb(95,107,166);
background-color:rgb(0,0,0);
border: 2px solid  rgb(95,107,166);
border-radius: 5px
}</string>
    </property>
    <property name="text">
     <string>Blink</string>
    </property>
   </widget>
   <widget class="QLabel" name="currBiasLbl">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>296</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#a8a8a8;&quot;&gt;Current Bias:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_readSetpoint">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed#push_readSetpoint{
color: rgb(95,107,166);
background-color:rgb(168,168,168);
border: 1px solid rgb(95,107,166);
border-radius: 5px
}

QPushButton#push_readSetpoint{
color: rgb(95,107,166);
background-color:rgb(0,0,0);
border: 2px solid  rgb(95,107,166);
border-radius: 5px
}</string>
    </property>
    <property name="text">
     <string>Read 
Set</string>
    </property>
   </widget>
   <widget class="Line" name="line_12">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>60</y>
      <width>244</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_9{color: rgb(168,168,168);}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_17">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>199</y>
      <width>50</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_9{color: rgb(168,168,168);}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_38">
    <property name="geometry">
     <rect>
      <x>27</x>
      <y>145</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#a8a8a8;&quot;&gt;Step Delay:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="BiasInc">
    <property name="geometry">
     <rect>
      <x>27</x>
      <y>110</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#a8a8a8;&quot;&gt;Number of Steps:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#a8a8a8;&quot;&gt;Go To Setpoint&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gotoBiasBtn">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>180</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed#gotoBiasBtn{
color: rgb(95,107,166);
background-color:rgb(168,168,168);
border: 1px solid rgb(95,107,166);
border-radius: 5px
}

QPushButton#gotoBiasBtn{
color: rgb(95,107,166);
background-color:rgb(0,0,0);
border: 2px solid rgb(95,107,166);
border-radius: 5px
}</string>
    </property>
    <property name="text">
     <string>Go To 
Bias</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="biasSetpntLine">
    <property name="geometry">
     <rect>
      <x>157</x>
      <y>75</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="biasDelayLine">
    <property name="geometry">
     <rect>
      <x>157</x>
      <y>145</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="currStatusLbl">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#a8a8a8;&quot;&gt;Status:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>145</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#a8a8a8;&quot;&gt;(ms)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line_11">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>61</y>
      <width>3</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_8{color: rgb(168,168,168)}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="push_Servers">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#push_Servers{
background: rgb(144, 140, 9);
 border-radius: 4px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Resources/nSOTScannerResources.qrc"/>
 </resources>
 <connections/>
</ui>
